apps/passport-server/src/util/zuzaluUser.ts

```diff
@@ -40,10 +40,16 @@ export function ticketsToMapByEmail(
  * Converts UserRow from database into ZupassUserJson to be returned from API.
  */
 export function userRowToZupassUserJson(user: UserRow): ZupassUserJson {
-  return {
-    uuid: user.uuid,
-    commitment: user.commitment,
-    email: user.email,
-    salt: user.salt
-  };
+  return Object.assign(
+    {
+      uuid: user.uuid,
+      commitment: user.commitment,
+      email: user.email,
+      salt: user.salt
+    } satisfies ZupassUserJson,
+    {
+      // TODO: remove this
```

Comment:
I understand this after your explanation, but others who see this might not. Maybe a more descriptive comment on why we want to remove this?

Reply:
Also, are we able to update forcibly update old clients, i.e. update the service worker file or version? If not, we may still need to keep this code.